spec.yaml

```diff
@@ -3112,6 +3115,55 @@ paths:
                       timestamp_start: "2011-08-30T00:00:00Z"
         '401':
            $ref: '#/components/responses/UnauthorizedError'
+  /meta/usage:
+    get:
+      tags:
+        - Usage
+      summary: API Usage Candles
```

Comment:
"API Key Usage" is a better name for people to find it. Also they aren't candles since they don't have OHLC.